pom.xml

```diff
@@ -38,12 +38,28 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-web</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
```

Comment:
We also have a `lombok` dependency in your `pom.xml`, between line `68` and `72`. So, we can remove this duplicate dependency tag.